src/components/SelectComponents/Autosuggest/Autosuggest.tsx

```diff
@@ -30,21 +30,32 @@ interface Props<S>
     inputPlaceholder: string;
     /** String to be shown when no suggestions are available */
     noSuggestionsPlaceholder?: string;
-    /** Defines if the first item of suggestions list is visible even while loading other elements */
-    isFirstItemAlwaysVisible?: boolean;
+    /** Defines if suggestions list is visible even while loading other elements */
+    allowMixingSuggestionsAndLoading?: boolean;
     /** Enable ListOptimizer component for decreasing render time */
     useOptimizeListRender?: boolean;
     /** Function to be called when a suggestion is removed  */
     onSelectionRemove?: (item: S) => void;
     /** Function to be called on submitting form */
     onSubmit?: () => void;
-    /** if suggestions are still loading, i.e. display placeholders */
+    /** If suggestions are still loading, i.e. display placeholders */
     isLoading?: boolean;
+    /** Means to overwrite the look and feel of the UI in its blurred state */
+    customBlurParams?: {
```

Comment:
It would be nice to have an example of using `customBlurParams` in storybook.

Reply:
There are 2 stories: `single select` and `dance as multi and single select`. You need to select something for it to be passed